operations/addresses.md

```diff
@@ -125,6 +127,7 @@ Adds one or more new addresses to the system and assign them to specified accoun
 | `ClientToken` | string | required | Token identifying the client application. |
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
+| `ChainId` | string | optional | Unique identifier of the chain. |
```

Comment:
these should be required for MP access token and actually they are not used for enterprise access token at all, it doesn't make sense for them. we should document it better. cc @MikeAdamsMews

Reply:
My suggestion would be simply "Unique identifier of the chain. This parameter is only needed when using [Portfolio Access Tokens](../guidelines/multi-property.md), in which case it is required."

Reply:
Also it would be nice if we had a place where a concept of chains is introduced and a way to retrieve ChainId value mentioned. Then we could link to it in comments like this.

Reply:
@jonas-paul chains are described under [Multi-property](https://mews-systems.gitbook.io/connector-api/guidelines/multi-property) \(this is the page linked to Portfolio Access Tokens\). But it's a good point about `ChainId` - where does the user get `ChainId` from?

Reply:
@MikeAdamsMews `ChainId` is on the `Enterprise` when you fetch `enterprises/getAll` or `configuration/get`